.github/workflows/ci.yaml

```
@@ -0,0 +1,32 @@
+name: Python Unit Tests with Coverage
+
+on:
+  push:
+    branches: [ unit_tests ]
+  pull_request:
+    branches: [ unit_tests ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.9' 
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
+          pip install coverage pytest

```

```suggestion
      - name: install poetry
        run: pip install poetry

      - name: install package
        run: poetry install --with test
```
Since the library is build with poetry, I would suggest using poetry here as well. The [pyrenew](https://github.com/CDCgov/PyRenew/blob/main/.github/workflows/test.yml) repo has a good example of using poetry for CI testing.

This will require adding `pytest` as a test dependency to the repo, by running `poetry add pytest --group test` and `poetry add coverage --group test`